test/inspector/test-async-stack-traces-promise-then.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper');
+const assert = require('assert');
+
+const script = 'Promise.resolve().then(() => { debugger; });';
```

Comment:
If you don't mind, I'd rather avoid filtering out async_hook events triggered for promises, simply because I don't see a reliable way how to detect them. The only async_hook information I have is a string-based `type`.

I wrote a simple program to verify what happens when both Node.js and V8 report asyncTask for the same promise:

```js
setInterval(() => {
  Promise.resolve().then(() => {
    debugger;
    console.log('tick')
  });
}, 300);
```

Things seems to work well with the current implementation, see the screenshot below (my test file is called `x.js`):

<img width="358" alt="screen shot 2017-07-13 at 18 24 04" src="https://user-images.githubusercontent.com/1140553/28176644-8058f8ee-67f8-11e7-94f5-6cca7976a10d.png">